main/tests/cypress/cypress/integration/project/grid/misc/expressions.spec.js

```diff
@@ -2,312 +2,286 @@
  * Utility method to load an expression panel, used by almost all the tests
  */
 function loadExpressionPanel() {
-	cy.columnActionClick('Shrt_Desc', ['Facet', 'Custom text facet'])
+  cy.columnActionClick('Shrt_Desc', ['Facet', 'Custom text facet']);
 }
 
 /**
  * Utility method to fill something into the expression input
  * Need to wait for OpenRefine to preview the result, hence the cy.wait
  */
 function typeExpression(expression) {
-	cy.get('textarea.expression-preview-code').type(expression)
-	cy.wait(250)
+  cy.get('textarea.expression-preview-code').type(expression);
+  cy.get('body[ajax_in_progress="false"]');
```

Comment:
@fgiroud Cypress discourages us to specify custom wait in the spec; any workaround which you could suggest me here as this is failing the test? Or should we ignore the lint for just this line 🤔 ?